backend/users/src/main/java/org/eclipse/sw360/users/UserHandler.java

```diff
@@ -47,16 +52,54 @@ public class UserHandler implements UserService.Iface {
 
     private static final Logger log = LogManager.getLogger(UserHandler.class);
     private static final String EXTENSION = ".log";
+    private final PasswordEncoder passwordEncoder = new BCryptPasswordEncoder(
+        12
```

Comment:
In other places the hashing rounds are set to default (10). May be we have to keep the same here for consistency or else we need to change in other places as well.

Reply:
I simplified it and made it default as the rest of the places.